addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -792,3 +795,9 @@ msgstr ""
 msgctxt "#31169"
 msgid "Artwork related settings."
 msgstr ""
+
+#. Label shown if subtitles are disabled
+#: /xml/Variables.xml
+msgctxt "#31170"
+msgid "Disabled"
+msgstr ""
```

Comment:
Missing new line as the end of the file

Reply:
Should be there now

Reply:
Why does you use https://github.com/xbmc/xbmc/blob/master/addons/resource.language.en_gb/resources/strings.po#L4237